src/phone-input/types.js

```diff
@@ -39,7 +41,8 @@ export type StateReducerT = (
 export type mapIsoToLabelT = (iso: string) => string;
 
 export type CountrySelectDropdownPropsT = {
-  children: $ReadOnlyArray<React.Node>,
+  // eslint-disable-next-line flowtype/no-weak-types
+  children: $ReadOnlyArray<React.Element<any>>,
```

Comment:
Should we keep the type here more loose to allow `null | string | number`?

Reply:
it needs to be a 'component' because of https://github.com/uber-web/baseui/blob/master/src/phone-input/country-select-dropdown.js#L56

referenced the flow docs here https://flow.org/en/docs/react/types/#toc-react-node

but, I am unsure about this typing. if it's a `div` element, it won't have props, right?